shared.go

```diff
@@ -285,7 +285,10 @@ func appendListedPackages(packages []string, mainBuild bool) error {
 			// like "arena", so older Go versions that we support
 			// do not yet have them and that's OK.
 			// Note that pkg.Standard is false for them.
+			// Note that Go 1.21 is swapping "GOROOT" for "std".
```

Comment:
why?

Reply:
GOROOT is a bit like GOPATH, in that most users shouldn't have to think about it anymore. So it's starting to be shown to the user less often.